### ISSUE TITLE
bug: `custom-video-speed`patch causes crashes.

### ISSUE THREAD

**Issue by VadimMillerVMMR** [inotia00/ReVanced_Extended#911]
### Type

Error at runtime

### Application

YouTube 18.16.39, revanced-patches 2.171.3.

### Bug description

YouTube app crashes if steps descripted in Steps to reproduce were made.

### Steps to reproduce

1) Make clean install;
2) Go to RVX Settings -> Video -> enable "Enable Custom Video Speed", restart the app;
3) Get back there, tap on "Default Video Speed" and select any option (except "auto");
4) When you try to enter the app next time, it will crash with no possibility to access.

### Relevant log output

```shell
java.lang.ExceptionInInitializerError
	at adpc.aq(PG:1)
	at adpc.I(PG:1)
	at adma.T(PG:7)
	at adpc.W(PG:1)
	at adma.P(PG:1)
	at iwa.cx(PG:119)
	at iwa.ck(PG:12)
	at iwa.aM(PG:5)
	at ivi.accept(PG:40)
	at j$.util.Optional.ifPresent(Unknown Source:4)
	at irq.a(PG:16)
	at aucq.tT(PG:2)
	at auuk.g(PG:15)
	at auuj.tT(PG:3)
	at auqd.g(PG:14)
	at aucp.tT(PG:3)
	at aucl.e(PG:3)
	at auxs.tR(PG:1)
	at auwr.tR(PG:6)
	at aunu.tR(PG:5)
	at aunu.tR(PG:5)
	at auwr.tR(PG:4)
	at auxj.tR(PG:1)
	at auwg.tR(PG:3)
	at auxj.tR(PG:1)
	at auwh.b(PG:4)
	at gfq.nf(PG:3)
	at ive.nf(PG:13)
	at ila.a(PG:48)
	at unz.b(PG:1)
	at afvu.b(PG:3)
	at agyj.run(PG:7)
	at android.os.Handler.handleCallback(Unknown Source:2)
	at android.os.Handler.dispatchMessage(Unknown Source:4)
	at android.os.Looper.loopOnce(Unknown Source:182)
	at android.os.Looper.loop(Unknown Source:82)
	at android.app.ActivityThread.main(Unknown Source:108)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(Unknown Source:11)
	at com.android.internal.os.ZygoteInit.main(Unknown Source:312)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'int java.lang.Integer.intValue()' on a null object reference
	at app.revanced.integrations.settings.SettingsEnum.getInt(SettingsEnum.java:381)
	at app.revanced.integrations.patches.utils.DoubleBackToClosePatch.<clinit>(DoubleBackToClosePatch.java:11)
	... 40 more
```

### Screenshots or videos

https://user-images.githubusercontent.com/127863693/235349093-75b35912-ce48-48e7-8b50-39eca18e161d.mp4

### Device Environment

Android 13, MIUI 14.

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by inotia00** [comment]
Thank you for attaching the crash log, fortunately I found the cause quickly

wait for the next release

**Comment by inotia00** [comment]
fixed in revanced-patches-v2.171.4
